lib/spack/docs/tutorial_modules.rst

```diff
@@ -367,19 +649,20 @@ Regenerating module files now we obtain:
 
 .. code-block:: console
 
-  $ spack module refresh --module-type tcl --delete-tree -y
+  root@module-file-tutorial:/# spack module refresh --module-type tcl --delete-tree -y
```

Comment:
A couple of lines above this (but not currently part of the PR) is "specify rules that applies", it should be "specify rules that apply".